bridges/Arte7Bridge.php

```diff
@@ -1,21 +1,48 @@
 <?php
 class Arte7Bridge extends BridgeAbstract {
 
-	// const MAINTAINER = 'mitsukarenai';
 	const NAME = 'Arte +7';
 	const URI = 'https://www.arte.tv/';
+	const MAINTAINER = 'imagoiq';
 	const CACHE_TIMEOUT = 1800; // 30min
 	const DESCRIPTION = 'Returns newest videos from ARTE +7';
 
 	const API_TOKEN = 'Nzc1Yjc1ZjJkYjk1NWFhN2I2MWEwMmRlMzAzNjI5NmU3NWU3ODg4ODJjOWMxNTMxYzEzZGRjYjg2ZGE4MmIwOA';
 
 	const PARAMETERS = array(
 		'global' => [
-			'video_duration_filter' => [
-				'name' => 'Exclude short videos',
+			'sort_by' => array(
+				'type' => 'list',
+				'name' => 'Sort by',
+				'required' => false,
+				'defaultValue' => 'broadcastBegin',
+				'values' => array(
+					'Video rights start date' => 'videoRightsBegin',
+					'Video rights end date' => 'videoRightsEnd',
+					'Brodcast date' => 'broadcastBegin',
+					'Creation date' => 'creationDate',
+					'Last modified' => 'lastModified',
+					'Number of views' => 'views',
+					'Number of views per period' => 'viewsPeriod',
+					'Available screens' => 'availableScreens',
+					'Episode' => 'episode'
```

Comment:
I got the values by sending a wrong value to the API, but the meaning is only inferred.